.github/workflows/superlinter.yml

```diff
@@ -0,0 +1,25 @@
+name: Super-Linter
+
+# Run this workflow every time a new commit pushed to your repository
+on: push
+
+jobs:
```

Comment:
**อันนี้ทำอะไรหว่า**
Almost done, @tanangular-hot! To complete your GitHub sign up, we just need to verify your email address: tanangular@hotmail.com.

Fullstack GraphQL: The Complete Guide to Writing GraphQL Servers and Clients with TypeScript -
Fullstack Graph'L is a complete guide to building servers and clients with Graph'L. Create robust, secure, and flexible APIs with Graph'L.

> Our second issue brings you five articles analyzing our climate’s past, present, and future. Deepen your understanding with these personalized and interactive data stories created by our talented authors

ลองดูตัวอย่างตามนี้นะ
[https://parametric.press/issue-02/](https://parametric.press/issue-02/)